relevanceai/dataset_api/dataset.py

```diff
@@ -137,6 +137,58 @@ def bulk_fn(documents):
             self.dataset_id, bulk_fn, select_fields=[self.field]
         )
 
+    def value_counts(
+        self,
+        normalize: bool = False,
+        ascending: bool = False,
+        sort: bool = False,
+        bins: Union[int, None] = None,
+    ):
+        schema = self.datasets.schema(self.dataset_id)
+        dtype = schema[self.field]
+
+        if dtype == "numeric":
+            agg_type = dtype
+        else:
+            agg_type = "category"
+
+        groupby_query = [{"name": self.field, "field": self.field, "agg": agg_type}]
+        aggregation = self.services.aggregate.aggregate(
+            self.dataset_id,
+            groupby=groupby_query,
+            page_size=10000,
+            asc=ascending,
+        )
+
+        total = self.get_number_of_documents(dataset_id=self.dataset_id)
+        aggregation = pd.DataFrame(aggregation)
+
+        if normalize:
+            aggregation["frequency"] /= total
+
+        if bins is not None:
+            vals = []
+            for agg in [[agg[0]] * int(agg[1]) for agg in aggregation.values]:
+                vals += agg
+
+            vals = pd.cut(vals, bins)
+
+            bins = [
```

Comment:
You've changed data type form integer to list. I'm not really sure why.

Reply:
easier for me to understand what is happening.